packages/website/ts/pages/not_found.tsx

```diff
@@ -11,15 +11,15 @@ export interface NotFoundProps {
     dispatcher: Dispatcher;
 }
 
-export const NotFound = (_props: NotFoundProps) => {
+export const NotFound = (props: NotFoundProps) => {
```

Comment:
What type does Visual Studio Code give this? I feel like it's just typed as (props: NotFoundProps) => React.ReactNode instead of React.StatelessComponent<NotFoundProps>.

Also (props: NotFoundProps) is technically incorrect because of the implied children props that is always present.

I still find it weird that it didn't catch this. Wonder if the explicit typing will catch it (I feel like it won't, I've run into runtime errors where I've done this by accident).

Reply:
Did you try React.ReactStatelessComponent?